firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/FirebaseCrashlytics.java

```diff
@@ -463,4 +463,23 @@ public boolean didCrashOnPreviousExecution() {
   public void setCrashlyticsCollectionEnabled(boolean enabled) {
     core.setCrashlyticsCollectionEnabled(enabled);
   }
+
+  /**
+   * Enables/disables/clears automatic data collection config by Crashlytics.
```

Comment:
Suggest replacing line 468 with:

`* Allows you to enable or disable the automatic data collection configuration in Crashlytics.`